src/components/banner.js

```diff
@@ -10,7 +10,7 @@ const Banner = () => {
           <span style={bannerStyles.infoIcon} className="material-symbols-outlined">
             info
           </span>
-          {bannerText}
+          <span className="banner-text" dangerouslySetInnerHTML={{ __html: bannerText }}></span>
```

Comment:
Hmmmm no, can we do something else here? Not a big fan of dangerously setting html